web-project/src/main/java/com/epam/webproject/dao/impl/BookDaoMock.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.webproject.dao.impl;
+
+import com.epam.webproject.dao.BookDao;
+import com.epam.webproject.domain.Book;
+import org.springframework.stereotype.Repository;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+@Repository
+public class BookDaoMock implements BookDao {
+
+    private Map<Integer, Book> storage = new HashMap<>();
+    private int id = 0;
```

Comment:
It's ok for mock dao, but don't forget that it's not safe to use `HashMap` and `int` for modification in multithreaded environment